Generator.md

```diff
@@ -0,0 +1,356 @@
+# Installation
+
+## Add the NuGet Package to your project
+
+All you need to do, to install Agoda.Ioc.Generator is to add a NuGet reference pointing to the package Agoda.Ioc.Generator.
+
+## PackageReference
+
+``` xml
+<PackageReference Include="Agoda.Ioc.Generator" Version="1.1.128"  PrivateAssets="all" />
+```
+
+## .NET CLI
+
+``` powershell
+dotnet add package Agoda.Ioc.Generator --version 1.1.128
+```
+
+**Note:** if you are using the .NET CLI. Please open `*.csproj` and add PrivateAssets="all" on PackageReference Include="Agoda.Ioc.Generator" Version="1.1.128" /> 
+
+**Note:** Please install the "Microsoft.Extensions.DependencyInjection.Abstractions" package if your project is not an asp.net project.
```

Comment:
This should be a depenecy of our library so this is automatic

Reply:
Added